dashboard/src/util/Loading.js

```diff
@@ -1,17 +1,13 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Dimmer, Loader, Segment } from 'semantic-ui-react';
 
-class Loading extends Component {
-  render() {
-    return (
+const Loading = ({message}) => (
         <Segment>
         <Dimmer inverted active>
-          <Loader inverted>{this.props.message || "Loading..."}</Loader>
+          <Loader inverted>{message || "Loading..."}</Loader>
         </Dimmer>
-        <div style={{"min-height":"3em"}}/>
+        <div style={{minHeight:"3em"}}/>
```

Comment:
For consistency you might want to use `styled` for this one. Remember emotion also provides a `css` template handler which simply generates a classname if you don't want to keep track of another component just for the CSS class.